source/placement_design.md

```diff
@@ -880,10 +880,9 @@ APPLICATIONNAME` operator command. All algorithms are aware of all constraints
 devices from other applications placed upon them. The available `ALGORITHM`s
 are:
 
- - `app`: See `bucket`.
+ - `app`: See `tfill`.
 
- - `bucket`: A bucket-filling placement, where the threads in the hardware
-   model are filled in sequence. This placement mechanism is device-type aware.
+ - `bucket`: See `tfill`.
```

Comment:
This section is missing an entry for the "link" synonym? 🍠 if not intentional, otherwise OK.